### PR TITLE
Feat app

### PR DESCRIPTION
**Pull request by null-harbor**

### CODE REVIEW

File: flo-server/index.js [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
-});
-
-app
-  .use(cors())
-  .use(Express.json())
-  .use(router)
-  .listen(PORT, () => {
-    console.log(`Server listen on port ${PORT} 🗺`);
-  });
+var Express = require('express');
```
Comment (irenecgn) [inline flo-server/index.js:1]: Why did you change everything with `var`?

Comment (null-harbor) [reply flo-server/index.js:1]: We chose your project for the legacy project. This was an accidental PR to your repo. I'm closing it now, we will only change the files from our fork.

Thanks! Sorry for the misunderstanding.

**Comment by null-harbor** [comment]
Accidental PR to main repo.
